providers/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -260,6 +260,8 @@ class KubernetesPodOperator(BaseOperator):
         "env_from",
         "node_selector",
         "kubernetes_conn_id",
+        "name",
+        "hostname",
```

Comment:
Can you please add tests that cover this change in test_pod.py?

Reply:
added, thanks!